test/e2e/framework/framework.go

```diff
@@ -378,7 +378,7 @@ func (f *Framework) CreateNamespace(baseName string, labels map[string]string) (
 		f.namespacesToDelete = append(f.namespacesToDelete, ns)
 	}
 
-	if !f.SkipPrivilegedPSPBinding {
+	if err == nil && !f.SkipPrivilegedPSPBinding {
```

Comment:
nit: just move this chunk into the if block above instead? (not sure if ns and err would ever both be nil)

Reply:
no, if err is non-nil, we're tearing down, not setting more stuff up